pkg/api/v1/helper/helpers.go

```diff
@@ -433,6 +433,28 @@ func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
 	return ""
 }
 
+// GetPersistentVolumeClaimVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeClaimVolumeMode(claim *v1.PersistentVolumeClaim) *v1.PersistentVolumeMode {
+	defaultMode := v1.PersistentVolumeFilesystem
+	if claim.Spec.VolumeMode != nil {
+		return claim.Spec.VolumeMode
+	}
+
+	return &defaultMode
+}
+
+// GetPersistentVolumeVolumeMode returns VolumeMode. If no VolumeMode was
+// requested, it returns "Filesystem" VolumeMode by default.
+func GetPersistentVolumeVolumeMode(volume *v1.PersistentVolume) *v1.PersistentVolumeMode {
```

Comment:
dtto, please return just `PersistentVolumeMode`

Reply:
@jsafrane - I fixed the helpers.go but need some eyes on the DisabledAlphaFields - I tried to follow as best I could, but not sure I did, can you take a look please?

@liggitt cc

Reply:
@screeley44, I think you got it right.